language/ir-testsuite/tests/move/generics/phantom_params/constraints_abilities_bad.mvir

```diff
@@ -0,0 +1,27 @@
+module M1 {
+    struct NoAbilities { a: bool }
+    struct HasAbilities<phantom T1, T2> has drop, copy, store, key { a: bool }
+}
+
+//! new-transaction
+
+module M2 {
+    import {{default}}.M1;
+
+    struct S1<T: drop + copy + store + key> { a: bool }
```

Comment:
There should also be a test for a single constraint

Reply:
done